README.md

```diff
@@ -188,6 +188,25 @@ dataset = StreamingInsideWebVid(local=local, remote=remote, shuffle=True)
 
 ---
 
+## Seamless data mixing
+
+Easily experiment with dataset mixtures with `Stream`. Dataset sampling can be controlled in relative (proportion) or absolute (repeat or samples terms). During streaming, the different datasets are streamed, shuffled, and mixed seamlessly just-in-time. 
```

Comment:
[nit] 
```suggestion
Easily experiment with dataset mixtures with [`Stream`](https://docs.mosaicml.com/projects/streaming/en/latest/api_reference/generated/streaming.Stream.html#stream). Dataset sampling can be controlled in relative (proportion) or absolute (repeat or samples terms). During training, the different datasets are streamed, shuffled, and mixed seamlessly just-in-time. 
```

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/266/commits/3c566cc6de4b159665e570970c13d5d2f4acad95.